tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -122,6 +122,7 @@
     - 'APC_PDU_CONTROLLER' - when APC PDU controller connected
     - 'NORDIC_BOARD_PDU_CONTOLLER' - when Nordic boards PDU controller connected
     - 'IP_POWER_SOCKET_PDU_CONTROLLER' - when IP Power Socket 5G10A connected
+    - 'EATON_PDU_CONTROLLER' - when EATON ODU controller connected
```

Comment:
typo ODU?
```suggestion
    - 'EATON_PDU_CONTROLLER' - when EATON PDU controller connected
```

Reply:
@doublemis1, can you confirm whether this is a typo?